### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Aquinasmoriasi**
- Setup application with create react app
- Removed boilerplate files
- Install configuration files and testing libraries
- Installed linters
- Created action reducers and creators to fetch and set data to the Redux state
- Set up the provider to enable access to the store by all components
- Added functional components to display the fetched data on the DOM
- Added styling to the various components
- Created a directory for all the test files
- Created test files
- Added tests for all the components and reducers

### CODE REVIEW

**Comment by Aquinasmoriasi** [comment]
- I had to create a pull request to the dev rather than to the main due to merge conflicts but the dev is still the base branch and all changes that will be merged to the dev shall then be merged to the main.

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,11 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Nav />
     </div>
```
Comment (smunozmo) [inline src/App.js:9]: - [ ] You have done an excellent job with this app, congrats! However, every item is using the same URL:

|1|2|
|---|---|
|Screen Shot 2022-09-09 at 6 05 14 PM|Screen Shot 2022-09-09 at 6 05 23 PM|

I would suggest using unique URLs for every item e.g: `/Details/1` `Details/2` etc, as this is a requirement for this activity.

Screen Shot 2022-09-09 at 6 07 12 PM

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @Aquinasmoriasi, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏 

- Another cool project for your portfolio, congrats on it! 🎉 
- You did an amazing job with the styling, great! 😎 
- Excellent job with the live demo and video demo, perfect! 💯 
- Tests are working properly, well done! ✅ 
- Redux is used correctly, excellent! 🤓 
- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Aquinasmoriasi) [reply src/App.js:9]: Thank you. It has been implemented.

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @Aquinasmoriasi,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Aquinasmoriasi** [comment]
Thank you.

**Pull request merged**